papertty/drivers/driver_it8951.py

```diff
@@ -188,21 +188,25 @@ def fixup_string(self, s):
         return result
 
     def init(self, **kwargs):
+
+        #SpiDev init must come first, otherwise CS_PIN read conflicts
+        #will sometimes occur during startup.
+        self.SPI = SpiDev()
+        self.SPI.setSpeed(2000000)
+        self.SPI.setMode(0b00)
+
         GPIO.setmode(GPIO.BCM)
         GPIO.setwarnings(False)
         GPIO.setup(self.RST_PIN, GPIO.OUT)
         GPIO.setup(self.CS_PIN, GPIO.OUT)
         GPIO.setup(self.BUSY_PIN, GPIO.IN)
```

Comment:
Now that these changes highlight it, it's a bit meh that this code is duplicated in `epd_init`, but that can be sorted out at a later time.

Reply:
Yeah, I noticed that. The only difference is the DC pin, which could just be a flag on the epd_reset method.
I might add that to this PR.

I need to fix something else here anyway.
There's the conflict in IT8951 and also I missed the SPI read functionality in IT8951 (the base driver doesn't implement that, only IT8951 does).
I'll try to get those fixed up tomorrow.

Reply:
I've added the epd_init change in and fixed the IT8951 conflict after rebasing the changes on joukos:main
I've also re-tested on both the IT8951 and on the EPD3in7 to confirm that both work.

Gonna need to do some work on VNC/FB mode to take full advantage of this, but terminal mode at least is working quite well.
On CPU-limited devices (I tried with a Raspberry Pi Zero WH) the performance difference is massive, even with the added network latency.